github_stargazers/github.py

```diff
@@ -99,6 +106,9 @@ def _check_hyperlink_component(component: element.Tag) -> None:
                 raise MissingHyperlinkTagError()
             if not hyperlink_component.get('href'):
                 raise MissingHrefAttributeError()
+            href_content: str = hyperlink_component['href']
+            if not re.match(r"/.+$", href_content):
```

Comment:
can you be more specific with the pattern matching?  how is the structure of the tag?

Reply:
Good observation, it wasn't very clear. 
I added a proper description to give it more context. PTAL over it.